linkml/generators/plantumlgen.py

```diff
@@ -129,13 +133,11 @@ def add_class(self, cn: ClassDefinitionName) -> str:
                 if True or cn in slot.domain_of:
                     mod = self.prop_modifier(cls, slot)
                     slot_defs.append(
-                        '    {field} "'
```

Comment:
I tend to add quotation marks to generated names for robustness: it doesn't harm having them and it won't break if any unexpected whitespaces appear in the names.

What does it harm to have them? If it doesn't harm, why removing them?

I suppose that we can rely on `underscore()` to ensure that no whitespaces appear in the names. So removing them shouldn't suppose any risk.

Reply:
Good questions @Silvanoc! I'd like to know the answer to the question you asked as well -- what are the pros of removing the quotes?